internal/server/server_test.go

```diff
@@ -238,196 +236,6 @@ func TestServer_Run_UIProxy(t *testing.T) {
 	})
 }
 
-func TestServer_GenerateRoutes_NoRoute(t *testing.T) {
```

Comment:
I don't think we want to lose this whole test. A couple of the test cases can be deleted, but at least 2 of them should still be relevant.

Reply:
Great catch. Will restore these